.github/workflows/ci.yml

```diff
@@ -189,9 +189,9 @@ jobs:
         with:
           push: true
           platforms: linux/amd64,linux/arm64
-          labels: "VERSION=${GITHUB_REF#refs/*/}"
+          labels: "VERSION=${{  github.ref_name }}"
           tags: |
-            quay.io/team-helium/miner:gateway-${GITHUB_REF#refs/*/}
+            quay.io/team-helium/miner:gateway-${{  github.ref_name }}
```

Comment:
Additionally, there's a unequal number of white spaces between `{{` (2) `github.ref_name` (1) `}}`

Reply:
fixed - good catch!